bot/orchestration/src/main/kotlin/ai/tock/bot/orchestration/bot/primary/RunOrchestrationStoryListener.kt

```diff
@@ -194,3 +203,26 @@ class RunOrchestrationStoryListener(
     private fun IntentAware?.inNoOrchestrationList(): Boolean = this?.wrappedIntent() in configuration.noOrchestrationIntentList
 
 }
+
+var BotBus.blockHandoverToSecondaryBot: Boolean
+    get() = userTimeline.userState.getFlag("block_handover_to_secondary_bot")?.toBoolean() ?: false
+    set(value) {
+        userTimeline.userState.setUnlimitedFlag("block_handover_to_secondary_bot", value.toString())
+    }
+
+var BotBus.blockTakeOverFromPrimaryBot: Boolean
+    get() = userTimeline.userState.getFlag("block_take_over_from_primary_bot")?.toBoolean() ?: false
+    set(value) {
+        userTimeline.userState.setUnlimitedFlag("block_take_over_from_primary_bot", value.toString())
+    }
```

Comment:
if bot A is primary of bot B and secondary of bot C, the developer must do a blockTakeOverFromPrimaryBot and a blockHandoverToSecondaryBot ?

Reply:
Maybe a global method of locking and unlocking function can help developers? ;-)

Reply:
- Great idea, finding a generic explicit name remains very difficult though

- I think all operations should be converted to functions if we are introducing global functions to ensure consistency(blockTakeOverFromPrimaryBot(), allowTakeOverFromPrimaryBot(), ...)